plugins/scaffolder-backend/src/scaffolder/actions/builtin/helpers.ts

```diff
@@ -237,6 +239,7 @@ export const enableBranchProtectionOnDefaultRepoBranch = async ({
           bypass_pull_request_allowances: bypassPullRequestAllowances,
           dismiss_stale_reviews: dismissStaleReviews,
         },
+        required_conversation_resolution: requiredConversationResolution,
```

Comment:
```suggestion
        require_conversation_resolution: requiredConversationResolution,
```
What do you think about naming it Require Conversation Resolution instead? I think it read a bit better.

Reply:
I totally agree that it reads better, but this name matches the underlying [API](https://github3.readthedocs.io/en/latest/api-reference/repos.html#github3.repos.branch.BranchProtection.required_conversation_resolution). The line suggested on must remain `required_conversation_resolution`, that's the name used in the client.

I could rename it everywhere else though. WDYT?

Reply:
I discussed it with the other maintainers and we opted to keep it `required_conversation_resolution` as it's closer to the github API 👍

Reply:
Thanks for the quick turnaround!